src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.scss

```diff
@@ -9,8 +9,9 @@
   display: grid;
   grid-template-areas:
     'left middle right'
-    'moreinformation . .'
-    'downloadLink . .'
+    'moreinformationLink . .'
+    'moreInformation moreInformation moreInformation'
```

Comment:
I think we should align CCDS  same as aa length as in xd

![Screenshot 2021-07-13 at 10 25 44](https://user-images.githubusercontent.com/6347854/125427578-74ae6a04-c373-4213-819a-b60d6b055994.png)

![Screenshot 2021-07-13 at 10 24 45](https://user-images.githubusercontent.com/6347854/125427383-2c97ecbc-22b3-433b-a3e9-549164dc0b7c.png)

Reply:
Also if you notice, the attributes on the first column have a left margin

Reply:
Good spot, i added a column gap (same as above)